classes/class.ilMumieTaskGradeListGUI.php

```diff
@@ -121,21 +119,14 @@ public function getAlert()
         return "";
     }
 
-
-
-    public function getPostVar()
-    {
-        return $this->postvar;
-    }
-
     /**
     * Get Post Variable.
     *
     * @return	string	Post Variable
     */
     public function getFieldId()
     {
-        $id = str_replace("[", "__", $this->getPostVar());
+        $id = str_replace("[", "__", null);
```

Comment:
`str_replace` on null will not result in a meaningful result.

The first argument is the needle you are looking for. The second is the value you want to replace the needle with. The third is the string you are searching,

Reply:
removed strreplace entirely and set return to null

Reply:
Do we really need a method to return null?

Reply:
no return value needed but the function itself is